### DISCUSSION TITLE
search in array column

### DISCUSSION THREAD

**Discussion by moinologics** [prest/prest#861] [API Server]
add support to search in an array column

example - 

table `users`

| id | name | blocked_user_ids |
|----|------|------------------|
| 1  | john | [3]              |
| 2  | jane | [1,3]            |
| 3  | bob  | []               |

resultant query

`SELECT name FROM users WHERE {{id}} = ANY(blocked_user_ids)`

when id = 1, return `jane`
when id = 3, return `john & jane`

**Comment by avelino** [comment]
Apparently the `ANY()` function has **not been implemented**, see here in the documentation


https://docs.prestd.com/api-reference/parameters#functions-support

@moinologics can you implement it? Would be here:

https://github.com/prest/prest/blob/9f4f28337bd2c1dbcb21691323ba160812b09fb0/adapters/postgres/postgres.go#L1531-L1550

**Comment by avelino** [comment]
```sql
SELECT name FROM users WHERE {{id}} = ANY(blocked_user_ids)
```

@moinologics to solve your problem without implementing the feature in the prestd core you can make custom queries, see the documentation [here](https://docs.prestd.com/api-reference/custom-queries)
